aten/src/ATen/native/native_functions.yaml

```diff
@@ -13985,12 +13992,18 @@
 - func: _chunk_grad_outputs_efficient_attention(Tensor query, Tensor key, Tensor value, bool is_causal=False) -> bool
   dispatch:
     CUDA: _chunk_grad_outputs_efficient_attention
-# Returns ouput, softmax_logsumexp, softmax
-- func: _flash_attention_forward(Tensor query, Tensor key, Tensor value, Tensor cum_seq_q, Tensor cum_seq_k, int max_q, int max_k, float dropout_p, bool is_causal) -> (Tensor, Tensor)
+# Returns ouput, softmax_logsumexp, rng_state
+- func: _flash_attention_forward(Tensor query, Tensor key, Tensor value, Tensor cum_seq_q, Tensor cum_seq_k, int max_q, int max_k, float dropout_p, bool is_causal) -> (Tensor, Tensor, Tensor)
```

Comment:
You can actually add names for these inline `-> (Tensor output, Tensor softmax_logsumexp, Tensor rng_State)`

Reply:
Same for derivatives. Might be good to label these outputs since there are a lot.

Reply:
WOW TIL